delorean/interface.py

```diff
@@ -35,7 +59,13 @@ def parse(s, dayfirst=True, yearfirst=True):
         # delorean object that represents the time.
         do = Delorean(datetime=dt, timezone=UTC)
     return do
-
+try:
+    parse.__doc__ = parse.__doc__.format(mode_utc=NAIVEMODE_UTC,
+                                         mode_raise=NAIVEMODE_RAISE,
+                                         mode_naive=NAIVEMODE_NAIVE)
+except AttributeError:
+    #must be running with -OO
```

Comment:
`python -OO` strips all docstrings.  If someone compiles delorean with `-OO` there would be no `parse.__doc__` and the code would fail without that handler.